src/picknik_dual_ur_arm_config/objectives/two_arms.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Two Arms">
+  <!-- ////////// -->
+  <BehaviorTree ID="Two Arms" _description="Move Two Arms" _favorite="true" _hardcoded="false">
```

Comment:
I would rename this so the Objective is called "Move Two Arms" and the description has a little more detail.